SchoolExplorer.IntegrationTest/Helpers/Utilities.cs

```diff
@@ -0,0 +1,54 @@
+﻿using SchoolExplorer.Domain.Entities;
+using SchoolExplorer.Infrastructure.MongoData;
+namespace SchoolExplorer.IntegrationTest.Helpers
+{
```

Comment:
This is terrible naming. 
please name and use classes as per encapsulation rules.